packages/core/src/types.ts

```diff
@@ -1826,6 +1826,7 @@ export interface ActorContext<
   TSystem extends ActorSystem<any> = ActorSystem<any>
 > {
   self: ActorRef<TEvent, TSnapshot>;
+  enqueue: (event: TEvent) => void;
```

Comment:
I think I like the idea of a privileged send function like this one. I also think that ideally, we should be able to route `done.invoke.*` events etc through a privileged channel like this.

Reply:
Great; I'll continue on with this